README.md

```diff
@@ -1,63 +1,33 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Youtube landing page
 
-# Project Name
-
-> One paragraph statement about the project.
+> trying to buile a page that works likeyoutube.
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+- html, css
+- vs code
 
-### Prerequisites
 
-### Setup
 
-### Install
+👤 **Ssegawa Alvin Louis K**
 
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/linkedinhandle)
+- Github: [@githubhandle](https://github.com/alvinlouis29)
+- Twitter: [@twitterhandle](https://twitter.com/louisssegawa)
+- Linkedin: [linkedin](https://linkedin.com/linkedinhandle/alvin-louis-632026183)
```

Comment:
This is an example inline comment. :slightly_smiling_face: 

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink: 

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.